<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_price</class>
 <widget class="QWidget" name="show_price">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="7,4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="show_sale">
     <property name="title">
      <string>正在出售</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTableWidget" name="sale_list">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="rowCount">
         <number>50</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>单价</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>品质</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>总价</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>服务器</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>雇员</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>更新时间</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>350</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>HQ图标</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>99</string>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string>35000</string>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string>紫水栈桥</string>
         </property>
        </item>
        <item row="0" column="5">
         <property name="text">
          <string>谁家雇员</string>
         </property>
        </item>
        <item row="0" column="6">
         <property name="text">
          <string>2022-5-10 16:06:04</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="show_min_price">
     <property name="title">
      <string>全服比价</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="rowCount">
         <number>8</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>服务器</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>单价</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>品质</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>总价</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>更新时间</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>紫水栈桥</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>延夏</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>静语庄园</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
